src/NuGet.Services.AzureSearch/SearchService/SearchTextBuilder.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    public class SearchTextBuilder : ISearchTextBuilder
+    {
+        public string V2Search(V2SearchRequest request)
+        {
+            var query = request.Query;
+
+            if (request.LuceneQuery)
+            {
+                // TODO: convert a leading "id:" to "packageid:"
```

Comment:
Do we have work items to address those TODOs? Can you put links/IDs into the comment itself?

Reply:
Great point! I'll add this.